### ISSUE TITLE
Upgrade from C117 -> C118 on Android ARM x64 via 1.60.x - Nightly (Checking C118)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#33232]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [ ] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [ ] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [ ] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers. We basically want to make sure that components are working as they regressed in C98 when we first merged into `master`.

### Upgrade

- [ ] Make sure that data from the last version appears in the new version OK
- [ ] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [ ] With data from the last version, verify that:
  - [ ] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [ ] Cookies are preserved
  - [ ] Installed extensions are retained and work correctly
  - [ ] Opened tabs can be reloaded
  - [ ] Stored passwords are preserved
  - [ ] Sync chain created in previous version is retained
  - [ ] Social media-blocking buttons changes are retained
  - [ ] Rewards
    - [ ] Wallet balance is retained
    - [ ] Auto-contribute list is retained
    - [ ] Both Tips and Monthly Contributions are retained
    - [ ] Wallet panel transactions list is retained
    - [ ] Changes to rewards settings are retained
    - [ ] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [ ] Ads
    - [ ] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [ ] Changes to ads settings are retained
    - [ ] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [ ] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by Uni-verse** [comment]
Verified on `Samsung Galaxy S21` using the following version(s):

```
Brave	1.60.47 Chromium: 118.0.5993.21 (Official Build) canary (32-bit) 
Revision	8c2edac2b59451745c32ee03f6aca51abfe4d222
OS	Android 13; Build/TP1A.220624.014; 33; REL
```

Example | Example
-|-
![Screenshot 2023-09-25 at 10 11 38 AM](https://github.com/brave/brave-browser/assets/17885425/3d899b80-b123-48de-8ed5-a45c3ab62ce0) | ![Screenshot 2023-09-25 at 10 14 54 AM](https://github.com/brave/brave-browser/assets/17885425/6152d560-d25a-463b-b624-86ccd60bc1f1)
